src/modules/common/Nav.jsx

```diff
@@ -9,48 +9,66 @@ export default class Nav extends React.Component {
 	};
 
 	render() {
-		const NavContainer = styled.nav`
-			text-align: center;
+		const Nav = styled.nav`
+			margin: 14px;
+			position: relative;
+			text-align: left;
+			z-index: 5;
 
 			a {
-				padding: 0 5px;
+				color: white;
+				position: relative;
+				display: inline-block;
+				font-size: 1.5rem;
+				font-weight: 700;
+				margin: 8px;
+				position: relative;
+
+				&:hover {
+					text-decoration: underline;
+				}
+
+				&:nth-child(2) {
+					float: right;
+				}
 			}
 		`;
 
-		return <NavContainer>{this._renderLinks()}</NavContainer>;
+		return(
+			<Nav>
+				{this._renderLinks()}
+			</Nav>
+		)
+
 	}
 
 	_renderLinks() {
-		return [...this._standardLinks(), ...this._authLinks()];
+		return this.props.user ? this._loggedInLinks() : this._loggedOutLinks()
 	}
 
-	_standardLinks() {
-		return [
-			<Link to={'/'} key={'home'}>
-				Home
-			</Link>,
-			<Link to={'/health'} key={'health'}>
-				Health
-			</Link>
-		];
+	_loggedOutLinks() {
+		return (
+			<>
+				<Link to={'/'} key={'home'}>
```

Comment:
High Priority:
Having string constants scattered through your code for routes is going to bite you in the ass.

Someday you're going to decide that home should be `'/me'`, and try to search through the places you can remember to update the links. But you'll miss one.

Instead, you should just have a global object with constants:
```js
const ROUTES = {
  HOME: '/',
  REGISTER: '/register',
  ...
}
```

Then anywhere you need a route, you just reference the object:
```jsx
<Link to={ROUTES.REGISTER} />
push(ROUTES.HOME)
```
Then when you decide to change home to `'/me'`, you change it in the route's object, and you're good to go.

Reply:
I was able to make this update!